code/modules/admin/quantum_mechanic.dm

```diff
@@ -0,0 +1,323 @@
+	spark_at(src)
+
+	QDEL_IN(src, 1 SECOND)
+
+	if(key)
+		var/mob/observer/ghost/ghost = new(src)	//Transfer safety to observer spawning proc.
+		ghost.key = key
+		ghost.dir = dir
+		ghost.mind.name = "[ghost.key] Quantum Mechanic"
+		ghost.name = "[ghost.key] Quantum Mechanic"
+		ghost.real_name = "[ghost.key] Quantum Mechanic"
+		ghost.voice_name = "[ghost.key] Quantum Mechanic"
+		ghost.admin_ghosted = TRUE
+		ghost.can_reenter_corpse = TRUE
+		ghost.reload_fullscreen()
+
+/mob/living/carbon/human/quantum/verb/quantum_antigrav()
+	set name = "Toggle Gravity"
+	set desc = "Toggles on/off falling for you."
+	set category = "Ω"
+
+	if (fall_override)
+		fall_override = FALSE
+		to_chat(usr, SPAN_NOTICE("You will now fall normally."))
+	else
+		fall_override = TRUE
+		to_chat(usr, SPAN_NOTICE("You will no longer fall."))
+
+/mob/living/carbon/human/quantum/verb/quantum_walk()
+	set name = "Ruin Everything"
+	set desc = "Uses bluespace technology to phase through solid matter and move quickly."
+	set category = "Ω"
+	set popup_menu = 0
+
+	if(usr.HasMovementHandler(/datum/movement_handler/mob/incorporeal))
+		usr.RemoveMovementHandler(/datum/movement_handler/mob/incorporeal)
+		to_chat(usr, SPAN_NOTICE("You will no-longer phase through solid matter."))
+	else
+		usr.ReplaceMovementHandler(/datum/movement_handler/mob/incorporeal)
+		to_chat(usr, SPAN_NOTICE("You will now phase through solid matter."))
+
+/mob/living/carbon/human/quantum/verb/quantum_recover()
+	set name = "Rejuv"
+	set desc = "Use the quantum within you to restore your health."
+	set category = "Ω"
+	set popup_menu = FALSE
+
+	revive()
+
+/mob/living/carbon/human/quantum/verb/quantum_awake()
+	set name = "Wake up"
+	set desc = "This is a quick fix to the relogging sleep bug."
+	set category = "Ω"
+	set popup_menu = FALSE
+
+	sleeping = 0
+
+/mob/living/carbon/human/quantum/verb/quantum_quit()
+	set name = "Teleport out"
+	set desc = "Activate bluespace to leave and return to your original mob (if you have one)."
+	set category = "Ω"
+
+	suicide(usr)
+
+/mob/living/carbon/human/quantum/verb/quantum_tgm()
+	set name = "Toggle Godmode"
+	set desc = "Enable or disable god mode. For testing things that require you to be vulnerable."
+	set category = "Ω"
+
+	status_flags ^= GODMODE
+	to_chat(usr, SPAN_NOTICE("God mode is now [status_flags & GODMODE ? "enabled" : "disabled"]."))
+
+// Bag o Holding
+/obj/item/storage/backpack/holding/quantum
+	storage_slots = 56
+	max_w_class = 400
+
+/obj/item/storage/backpack/holding/quantum/attack_hand(mob/user)
+	if(!user)
+		return
+
+	if(!istype(user, /mob/living/carbon/human/quantum))
+		to_chat(user, SPAN_WARNING("Your hand seems to go right through the [src]. It's like it doesn't exist."))
+		return
+
+	return ..()
+
+// Headset
+/obj/item/radio/headset/ert/quantum
+	name = "quantum mechanic's headset"
+	desc = "A Quantum Mechanic's headset. The letter 'Ω' are stamped on the side."
```

Comment:
Lowercase on `quantum mechanic` for consistency. Also `the letter is` rather than `are`.